.github/workflows/playwright.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Download blob reports from GitHub Actions Artifacts
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4.1.7
```

Comment:
@kyranjamie: I was trying to update this as it was recommended in https://github.com/leather-io/extension/pull/5841. 

It gives this CI error though:
![Screenshot 2024-09-04 at 09 18 53](https://github.com/user-attachments/assets/6c67119d-856c-4d07-9827-e7ba9d992bc1)

Maybe we don't need to do it and can ignore?

Reply:
Mmm I see here's it'll be deprecated so might be good to get fixed

![image](https://github.com/user-attachments/assets/413364e9-a5a0-4112-ae8f-ed841ff9ac30)

There's a migration guide https://github.com/actions/download-artifact/blob/main/docs/MIGRATION.md — I wonder if we need to name them differently 🤔

Reply:
Thanks. It seems non trivial to do it so I opened [this issue](https://github.com/leather-io/extension/issues/5845) to schedule migration. 

I'll drop the update from here so we can update webpack